docs/getting-started.md

```diff
@@ -254,8 +272,10 @@ scanner on the `dependencies` scope in this tutorial. If you also want to scan t
 advised to configure a cache for the scan results as documented in the [README](../README.md) to speed up repeated
 scans.
 
+As "duing" the _analyzer_ step an `.ort.yml` configuration file was provided to exclude `devDependencies`, the `--skip-excluded` option can be used to avoid the download and scanning of that scope.
```

Comment:
I believe you misunderstood my comment, I wanted to point out the type in `duing`, it should be `during` (without the quotes).

Reply:
yeah 🤦‍♂ , and i was wondering why you wanted to emphasize during. fixed